docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -260,8 +271,125 @@ Server can set age rating for current playback. Client can access this rating (r
     }
     ```
 
+## Managing playlists on server side
+
+To manage the media controller playlists in your server application, you must learn to create, save, delete playlists.
+
+1. Create a media controller playlist using the `createPlaylist()` method:
+
+   ```
+   var playlist = mcServer.createPlaylist("testPlaylistName");
+   ```
+
+2. Adding item to playlist is available using `addItem()` method:
+
+   ```
+   var metadata = {
+       title: "testTitle",
+       artist: "testArtist",
+       album: "testAlbum",
+       author: "testAuthor",
+       genre: "testGenre",
+       duration: "testDuration",
+       date: "testDate",
+       copyright: "testCopyright",
+       description: "testDescription",
+       trackNum: "testTrackNum",
+       picture: "testPicture"
+   };
+
+   playlist.addItem("index1", metadata);
+   ```
+
+3. To save playlist use `savePlaylist()` method:
+
+   ```
+   function successCallback() {
+       console.log("savePlaylist successful.");
+   }
+
+   function errorCallback(error) {
+       console.log("savePlaylist failed with error: " + error);
+   }
+   mcServer.savePlaylist(successCallback, errorCallback);
+   ```
+
+4. To get information about playlists use `getAllPlaylists()` method:
+
+   ```
+   function successCallback(playlists) {
+       playlists.forEach(function(playlist) {
+           console.log("Playlist name: " + playlist.name);
+       });
+   }
+   function errorCallback(error) {
+       console.log("getAllPlaylists failed with error: " + error);
+    }
+   mcServer.getAllPlaylists(successCallback, errorCallback);
+   ```
+
+5. To get information about playlist items use `getItems()` method:
```

Comment:
Suggest rephrasing to:
5. Get information about the playlist items using the `getItems()` method:

Reply:
Global comment, please add "the":

5. To get information about playlist items use `getItems()` method: